### DISCUSSION TITLE
details of the getstatus command

### DISCUSSION THREAD

**Discussion by nickscut** [lf-edge/ekuiper#1077] [Q&A]
# bin/kuiper getstatus rule rule1
{
    "source_demo_0_records_in_total":5,
    "source_demo_0_records_out_total":5,
    "source_demo_0_exceptions_total":0,
    "source_demo_0_process_latency_ms":0,
    "source_demo_0_buffer_length":0,
    "source_demo_0_last_invocation":"2020-01-02T11:28:33.054821",
    ... 
    "op_filter_0_records_in_total":5,
    "op_filter_0_records_out_total":2,
    "op_filter_0_exceptions_total":0,
    "op_filter_0_process_latency_ms":0,
    "op_filter_0_buffer_length":0,
    "op_filter_0_last_invocation":"2020-01-02T11:28:33.054821",
    ...
}

**Is there a document describing the meaning of each field?**

**Comment by ngjaying** [comment]
There is no specific doc. There are actually quite self explainary. Each rule will be parsed and run as a DAG of nodes.  source/op/sink is the type of the node, demo or filter is the name of the node.
